keps/sig-node/2400-node-swap/README.md

```diff
@@ -1059,6 +1169,7 @@ nodes that do not use swap memory.
 - **2021-01-05:** Initial design discussion document for swap support and use cases.
 - **2021-04-05:** Alpha KEP drafted for initial node-level swap support and implementation (KEP-2400).
 - **2021-08-09:** New in Kubernetes v1.22: alpha support for using swap memory: https://kubernetes.io/blog/2021/08/09/run-nodes-with-swap-alpha/
+- **2024-01-12:** Updates to Beta2.
```

Comment:
Would it make sense to add a reference to the PR that added swap support in the stats summary? https://github.com/kubernetes/kubernetes/pull/118865

Reply:
Although, I see that  section only has links to issues, not PRs.

Reply:
This section does need some love. I don't really know the dates but I guess I can find PRs some I can fill this in.

Reply:
I updated the section with what I could find. There are alot of PRs and I decided to focus on promotion to beta1. PTAL.